azext_iot/monitor/parsers/common_parser.py

```diff
@@ -84,6 +86,14 @@ def _parse_device_id(self, message: Message) -> str:
             self._add_issue(severity=Severity.error, details=details)
             return ""
 
+    def _parse_module_id(self, message: Message) -> str:
+        try:
+            return str(message.annotations.get(MODULE_ID_IDENTIFIER), "utf8")
+        except Exception:
+            details = strings.unknown_module_id()
```

Comment:
I'd suggest not logging the module name as an error, because non-edge devices should NOT be sending a module name.

I'd suggest just "passing" here, similar to how parsing interface is currently done